app/routes/games/list.js

```diff
@@ -0,0 +1,28 @@
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  actions: {
+    newGame: function () {
+      // The user must not be anonymous.
+      if (this.get('session.user.isAnonymous')) {
```

Comment:
Should we also disallow a user from creating a game if they don't have at least one deck that's ready for play? We could add a computed property to the deck for determining if ready to play (mainly just checking for at least 60 cards) and then verify the user has at least one of those.